OsmAnd/res/values/strings.xml

```diff
@@ -661,9 +661,9 @@
     <string name="srtm_purchase_header">To see contour lines on the map, you need to buy and install the \'Contour Lines\' plugin</string>
     <string name="srtm_color_scheme">Color scheme</string>
     <string name="show_from_zoom_level">Display from zoom level</string>
-    <string name="routing_attr_allow_private_name">Allow private access</string>
+    <string name="routing_attr_allow_private_name">Allow trespassing</string>
```

Comment:
You suggested this type of change before. "trespassing" is not a good word to use here. The option is to allow access to private roads. For example, suppose you're staying at a hotel and it has a private access road. As a guest of the hotel, you have permission (perhaps implied, but nevertheless permission) to use the road, and therefore it is not trespassing.

Reply:
Agree, private roads is quite easy to understand, while trespassing is something related to violation

Reply:
Thought I might have. It is a foreign concept to me, where I live a private road is not private  until there is a gate on it.
I don't think it is always a question of being offroad when using small paths, (not roads) some of which may be private ones?

Reply:
'Trespassing' is wrong here.